### PR TITLE
#3 standardize aws lex endpoint

### LINKED ISSUE #3: Standardize aws-lex-endpoint
This is to select an official endpoint and perform standardization.

Remember to add two or three important business flow steps.

### PR DESCRIPTION
**Pull request by agreggio-slingr**
Pull request for the topic https://github.com/slingr-stack/aws-lex-endpoint/issues/3 created by @agreggio-slingr

## What does it do?
This ticket implements the use of slingr-helpgen, whereby it modifies existing functions, adds basic functions and step flows.

## How to test it?
Make calls from the platform for example using the console using the information provided in the documentation.

## Implementation notes
Create new version of the endpoint, it is not compatible with previous versions.

## Implementation notes
Create a new tags changing the major version and deploy to the platform.

### CODE REVIEW

**Comment by sebasmoyano** [comment]
#approve

File: flowSteps/createBotVersionAwsLex/step.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "label": "Create Bot Version",
+  "name": "createBotVersionAwsLex",
+  "category": "integrations",
+  "description": "Creates a new version of the bot based on the $LATEST version. If the $LATEST version of this resource hasn't changed since you created the last version, Amazon Lex doesn't create a new version. It returns the last created version.",
+  "inputs": [
+    {
+      "label": "Bot Name",
+      "name": "botName",
```
Comment (pasaperez-work) [inline flowSteps/createBotVersionAwsLex/step.json:9]: Check the name of the "Bot Name" entry, it has to be the same as the script.

File: flowSteps/deleteBotAwsLex/step.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "label": "Delete Bot",
+  "name": "deleteBotAwsLex",
+  "category": "integrations",
+  "description": "Deletes all versions of the bot, including the $LATEST version. To delete a specific version of the bot, use the DeleteBotVersion operation. The DeleteBot operation doesn't immediately remove the bot schema. Instead, it is marked for deletion and removed later.",
+  "inputs": [
+    {
+      "label": "Bot Name",
+      "name": "botName",
```
Comment (pasaperez-work) [inline flowSteps/deleteBotAwsLex/step.json:9]: Check the name of the "Bot Name" entry, it has to be the same as the script.

File: flowSteps/getBotVersionAwsLex/step.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
+  "label": "Get Bot Version",
+  "name": "getBotVersionAwsLex",
+  "category": "integrations",
+  "description": "Returns metadata information for a specific bot. You must provide the bot name and the bot version or alias.",
+  "inputs": [
+    {
+      "label": "Bot Name",
+      "name": "botName",
```
Comment (pasaperez-work) [inline flowSteps/getBotVersionAwsLex/step.json:9]: Check the name of the "Bot Name" entry, it has to be the same as the script.

File: README.md [hunk 4]
```diff
@@ -129,3 +129,659 @@ SLINGR is a low-code rapid application development platform that accelerates dev
 ## License
```
Comment (pasaperez-work) [inline README.md]: According to the general format, we are leaving the "# About SLINGR" and "# License" sections at the end of the Readme.

File: endpoint.json [hunk 5]
```diff
@@ -41,6 +61,12 @@
         "functions.js",
         "lex-helpers.js"
```
Comment (pasaperez-work) [inline endpoint.json:62]: Can we unify the lex-helpers.js helpers with the funtions.js?
This way we could use only the "_{method}" and remove the old "_http{Method}".

**Review by pasaperez-work — CHANGES_REQUESTED** [review]

File: endpoint.json (hunk 5 above)
Comment (agreggio-slingr) [reply endpoint.json:62]: Leave the previous helpers and do not remove the methods to support backwards compatibility.

**Review by pasaperez-work — APPROVED** [review]

**Comment by pasaperez-work** [comment]
#approve

**Comment by pasaperez-work** [comment]
Remember to create a new tag for version release.

**Comment by agreggio-slingr** [comment]
> Remember to create a new tag for version release.

created

**Pull request merged**